### PR TITLE
Examples in Gambas

### PR DESCRIPTION
**Pull request by vdelacruzjardon**
Hello

I am working on the examples in Gambas Basic language.
http://gambas.sourceforge.net/en/main.html

Thanks

### CODE REVIEW

File: 01_introduction_to_algorithms/gambas/01_binary_search/.src/binary_search.module~ [hunk 1]
```diff
@@ -0,0 +1,28 @@
+' Gambas module file
```
Comment (egonSchiele) [inline 01_introduction_to_algorithms/gambas/01_binary_search/.src/binary_search.module~]: Can you delete files that end with "~"? These are temporary files.

File: 01_introduction_to_algorithms/gambas/01_binary_search/.settings [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[Breakpoints]
```
Comment (egonSchiele) [inline 01_introduction_to_algorithms/gambas/01_binary_search/.settings]: Can you delete all settings files?

File: 01_introduction_to_algorithms/javascript/01_binary_search.js [hunk 3]
```diff
@@ -1,26 +0,0 @@
-'use strict';
```
Comment (egonSchiele) [inline 01_introduction_to_algorithms/javascript/01_binary_search.js]: You can't delete these files

File: 02_selection_sort/gambas/01_selection_sort/.directory [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[Desktop Entry]
```
Comment (egonSchiele) [inline 02_selection_sort/gambas/01_selection_sort/.directory]: Can you delete these .directory files and the associated icons?

**Comment by egonSchiele** [comment]
Thank you very much! Your PR looks great, it just needs some cleanup before I can merge it. I left some comments about what to clean up. Could you also squash your commits into one commit?

**Comment by vdelacruzjardon** [comment]
Sorry I am very newbie in Git . I have made the corrections but I dont know if iws squash correctlly. Thanks

**Comment by egonSchiele** [comment]
No problem! I will take a look soon.

**Comment by egonSchiele** [comment]
Merged! Thanks.

**Pull request merged**